src/components/prompt-dialog/PromptDialog.tsx

```diff
@@ -69,7 +65,7 @@ export default function PromptDialog({ open, onSubmit, onCancel, isLoading, setI
           >
             {isLoading ? (
               <>
-                <Loader2 className="w-4 h-4 mr-2 animate-spin" />
+                <Loader2 aria-label="loader" className="w-4 h-4 mr-2 animate-spin" />
```

Comment:
Likely "Loading" is a better `aria-label` in this situation, but I love that you noticed it!

Reply:
Although since we already have `Generating...` below, it would be even better if this was just ignored by screen-readers with `aria-hidden={true}`.

Reply:
I understand what you mean, but the word Generating... only appears when it is loading. At the beginning, it is not loading, and generate does not indicate what it is going to generate.

Reply:
> but the word Generating... only appears when it is loading

The `aria-label` too, right?

> and generate does not indicate what it is going to generate

Neither does `loader`, but it's one less thing to maintain. In general, when descriptive text can be used for a11y, I prefer it over `aria-label`s... I'd even prefer adding a span with `sr-only` which describes what's generating better.

---

Anyway, this is minor and what you suggested already improves stuff!